### PR TITLE
add an example contract ContractAsLP.sol

### PR DESCRIPTION
**Pull request by ClaudeZsb**

### CODE REVIEW

**Comment by xiaozhou** [comment]
Put it in test-helpers folder

**Comment by ClaudeZsb** [comment]
> 

ok

**Comment by ClaudeZsb** [comment]
> Put it in test-helpers folder

ok

File: contracts/test-helpers/IPool.sol [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0-only
```
Comment (xiaozhou) [inline contracts/test-helpers/IPool.sol:1]: Put these two interfaces in the interface folder

Comment (ClaudeZsb) [reply contracts/test-helpers/IPool.sol]: ok

**Review by xiaozhou — APPROVED** [review]

**Pull request merged**